linera-service/src/linera/main.rs

```diff
@@ -1448,6 +1459,110 @@ impl ClientOptions {
     }
 }
 
+struct DatabaseToolJob<'a>(&'a DatabaseToolCommand);
+
+#[async_trait]
+impl RunnableWithStore for DatabaseToolJob<'_> {
+    type Output = i32;
+
+    async fn run<S>(
+        self,
+        config: S::Config,
+        namespace: String,
+    ) -> Result<Self::Output, anyhow::Error>
+    where
+        S: KeyValueStore + Clone + Send + Sync + 'static,
+        S::Error: Send + Sync,
+    {
+        match self.0 {
+            DatabaseToolCommand::DeleteAll => {
+                let start_time = Instant::now();
+                S::delete_all(&config).await?;
+                info!(
+                    "All namespaces deleted in {} ms",
+                    start_time.elapsed().as_millis()
+                );
+            }
+            DatabaseToolCommand::DeleteNamespace => {
+                let start_time = Instant::now();
+                S::delete(&config, &namespace).await?;
+                info!(
+                    "Namespace deleted in {} ms",
+                    start_time.elapsed().as_millis()
+                );
+            }
+            DatabaseToolCommand::CheckExistence => {
+                let start_time = Instant::now();
+                let test = S::exists(&config, &namespace).await?;
+                info!(
+                    "Existence of a namespace checked in {} ms",
+                    start_time.elapsed().as_millis()
+                );
+                if test {
+                    println!("The database does exist");
+                    return Ok(0);
+                } else {
+                    println!("The database does not exist");
+                    return Ok(1);
+                }
+            }
+            DatabaseToolCommand::CheckAbsence => {
+                let start_time = Instant::now();
+                let test = S::exists(&config, &namespace).await?;
+                info!(
+                    "Absence of a namespace checked in {} ms",
+                    start_time.elapsed().as_millis()
+                );
+                if test {
+                    println!("The database does exist");
```

Comment:
Would it be better to print something more machine-readable, and log this as `info!` instead?